mlabs/src/Mlabs/Nft/Contract/StateMachine.hs

```diff
@@ -132,30 +132,20 @@ nftValue nid = assetClassValue (nftCoin nid) 1
 ------------------------------------------------------------------------
 
 runStepWith :: forall w e schema .
-  ( SM.AsSMContractError e
-  , HasUtxoAt schema
```

Comment:
These now obsolete. As an example of how changes should be made I took `retrieveFundsC` in `Vesting.hs` of `plutus-use-cases` of `plutus` repository for current version.